model_prices_and_context_window.json

```diff
@@ -1,10 +1,10 @@
 {
     "gpt-4": {
-        "max_tokens": 4096, 
+        "max_tokens": 4096,
         "max_input_tokens": 8192,
-        "max_output_tokens": 4096, 
-        "input_cost_per_token": 0.00003,
-        "output_cost_per_token": 0.00006,
+        "max_output_tokens": 4096,
```

Comment:
please do not edit OpenAI pricing in this PR or with your script. It should only be limited to Open router

Reply:
Hi @ishaan-jaff , I double checked: we are only updating the values of open-router.

Following are **same values**, one in `float` another in `scientific notation`.
<img width="1300" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/6441d145-0ae1-47bc-be1b-3ea9eec84b23">

This is a default behavior of the `json` package, that converts the float to "scientific notation".

If we want the floating point not "scientific notation" then, the **floating point values will be string type** similar to what OpenRouter is doing `"0.000001125"`:
<img width="663" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/5307c504-5ebc-4d7a-83d1-8283c102f2bb">

> Because of floating point precision `json` package uses scientific notation.